### ISSUE TITLE
Pictures for Readme

### ISSUE THREAD

**Issue by MatthieuVeillon** [MatthieuVeillon/quokka-test#1]
![output global json](https://user-images.githubusercontent.com/20015180/32003528-6041305a-b96d-11e7-9c21-8ba332e43c41.png)
![output snippet file](https://user-images.githubusercontent.com/20015180/32003529-605035d2-b96d-11e7-83fd-f3a02d99cbea.png)

**Comment by ArtemGovorov** [comment]
Thanks for sharing the repo!

Unfortunately I still can't reproduce the issue. It's working for me on the same OS/editor:

<img width="649" alt="screen shot 2017-10-26 at 11 33 13 am" src="https://user-images.githubusercontent.com/979966/32031219-3ac184ca-ba43-11e7-9ed4-79fff230c21d.png">

Could you please open VS Code terminal (Ctrl + `), run these:
```
> node
> require('path').resolve(require('path').join('.', 'node_modules', 'babel-core'))
```

and paste the result?

Also try specifying the path to babel right in the config:

```diff
({
  babel: {
+   path: 'full_path_to_your_projec_dir/node_modules/babel-core',
    presets: ["react-app"]
  }
});
```

**Comment by MatthieuVeillon** [comment]
Here you go, didn't change anything unfortunately.

```
> require('path').resolve(require('path').join('.', 'node_modules', 'babel-core'))
'/Users/matthieuveillon/GoogleDrive/LearnCode/node_modules/babel-core'
```

![app jsx](https://user-images.githubusercontent.com/20015180/32056437-af5a8ec8-ba32-11e7-88fd-7fcec89deafd.png)


Let me also copy paste my config file from vscode, could some of my settings interfere with quokka ? 

`{
  "editor.formatOnSave": true,
  "editor.fontFamily": "Fira Code",
  "editor.fontLigatures": true,
  "vscode_custom_css.imports": [
    "file:///Users/matthieuveillon/fontstyle/style.css"
  ],
  "workbench.colorTheme": "Monokai Pro (Filter Octagon)"
}
`

**Comment by ArtemGovorov** [comment]
Thanks for providing the details.

Could you please open VS Code terminal (Ctrl + `), run these:
```
> node
> require(require('path').resolve(require('path').join('.', 'node_modules', 'babel-core'))).version
```

**Comment by ArtemGovorov** [comment]
Just in case, could you please try moving the repo somewhere else, like `/Users/matthieuveillon/Documents` and trying from there?

Also, please create a new Quokka file (`Cmd/Ctrl + K, J`), paste `console.log(process.env)` into it, and paste the output from Quokka (not that you may need to remove some security sensitive keys from it if you have some there).

**Comment by MatthieuVeillon** [comment]
Hello @ArtemGovorov - sry for the delay.

1) on the initial project : 
> require(require('path').resolve(require('path').join('.', 'node_modules', 'babel-core'))).version
'6.25.0'

2) when I move the repo to an other folder like `/Users/matthieuveillon/Documents` then I get something weird : 

_Only 'Pro' edition of Quokka.js supports project files import_ 

3) when I create the file and console.log process.env it seems to work - see picture below 
![capture d ecran 2017-11-06 a 20 11 51](https://user-images.githubusercontent.com/20015180/32472163-36a37a66-c32f-11e7-8458-a58a8e8f8294.png)

**Comment by ArtemGovorov** [comment]
> when I move the repo to an other folder like /Users/matthieuveillon/Documents then I get something weird


This is actually expected, so now it's working as expected and correctly compiles everything. So I think the issue is the location of the project in the `GoogleDrive` folder of yours (perhaps it's symlinked in some way or something else, but Quokka or node resolution algo doesn't like it).

> `Only 'Pro' edition of Quokka.js supports project files import`

It is an expected message. You are using Community edition of Quokka and only [Pro version supports project files imports](https://quokkajs.com/pro/#project-files). You may switch to Pro edition to trial it if you like (by using the prompt you should see with the message, or by adding the `"pro": true` to your `Users/matthieuveillon/.quokka/config.json` file).

**Comment by MatthieuVeillon** [comment]
Thanks for the fast answer - help much appreciated. I confirm that switching locations make it work for all the files / projects

**Comment by MatthieuVeillon** [comment]
Hey @ArtemGovorov, sorry to bother you again. 

But the inconsistent (babel-core module not found while it's installed) behaviour is coming back. I tried different locations on my computer (documents, desktop, Users) and it's rather inconsistent. Therefore I assume the sync from google or whatever is not the culprit. 

New CRA file or existing ones have the same results

When I do a path check I get the usual output below 
> require('path').resolve(require('path').join('.', 'node_modules', 'babel-core'))
'/Users/matthieuveillon/Documents/quokka-test/node_modules/babel-core'

See also screen below. 

If you want I can move the issue on your github as an issue to share the information with anyone concerned.

Thanks in advance 

![capture d ecran 2017-11-09 a 09 59 40](https://user-images.githubusercontent.com/20015180/32612033-2b5c2966-c535-11e7-862d-18dd02a8e114.png)

**Comment by ArtemGovorov** [comment]
> and it's rather inconsistent

Do you mean it sometimes works, and sometimes doesn't?

We have also just extended the error message to provide some more details, please update to the the latest v1.0.77 and share the new error message from the Quokka console.

**Comment by MatthieuVeillon** [comment]
Ok your error log helped. 

I have a project in my document folder. Following path :

`/Users/matthieuveillon/Documents/CodeProjects/quokka-test`

Whenever I open Vscode from a parent folder  (let's say CodeProjects)  navigate through my file and start quokka on it - I get the following errors message : 

![capture d ecran 2017-11-09 a 20 58 34](https://user-images.githubusercontent.com/20015180/32639202-b52c84f0-c590-11e7-92c6-f181305371fa.png)

On the other end when I directly open my CRA folder as the root in vscode, quokka looks for the file at the right place. Which led me to believe to the inconsistent behaviour. 

![capture d ecran 2017-11-09 a 20 57 11](https://user-images.githubusercontent.com/20015180/32639206-ba8aa684-c590-11e7-98ca-c91d88dfffae.png)

Aside from this path related issue. When I properly launch it on my current project. I manage to get the output in the output console. But not in my code. I get "Language "wallaby-output" is not supported. screen below. Any idea on the causes for that ?  
Also the bottom right loading sign never stops. 


![capture d ecran 2017-11-09 a 21 05 02](https://user-images.githubusercontent.com/20015180/32639359-923e86e0-c591-11e7-980d-2a036069cbb2.png)



Thanks,

**Comment by ArtemGovorov** [comment]
>  I get "Language "wallaby-output" is not supported. screen below. Any idea on the causes for that ?

When you're using the Quokka start command, it tries to start on a currently focused document. In this case the currently focussed document is the output. If you switch the focus to your main editor and start Quokka, then it should work as expected.

**Comment by MatthieuVeillon** [comment]
makes sense. I don't get the language output error when I follow your instructions, but I still don't get the output in the main editor. I assume it has to do with my code (and the spinner still going on) - are there any general case for which quokka cannot resolve the code in the editor ? specifically in react ? 

file here : https://github.com/corscheparrera/Alibay-project/blob/front-end-React/src/Components/jsx/AccountPage.jsx

thanks for all your help on this.

**Comment by ArtemGovorov** [comment]
@MatthieuVeillon spinner normally will be going on until the execution is finished. The easiest way to troubleshoot is by removing parts of the code until it runs. If you can update the repo with some sample that I can run, I'm happy to have a look into.
